tutorials/plot_events_and_annotations.py

```diff
@@ -0,0 +1,120 @@
+"""
+The **events** and :class:`Annotations <mne.Annotations>` data structures
+=========================================================================
+
+Events and annotations are quite similar. This tutorial highlights their
+differences and similitudes and tries to shade some light to which one is
+preferred to use in different situations when using MNE.
+Here follows both terms definition from the glossary.
+
+    events
+        Events correspond to specific time points in raw data; e.g., triggers,
+        experimental condition events, etc. MNE represents events with integers
+        that are stored in numpy arrays of shape (n_events, 3). Such arrays are
```

Comment:
I am always forget what the second column of the events array is. Maybe it's worth repeating here what the three columns represent?

Reply:
For this you need an example where the second column is more than array of 0s. And is not yet covered in this tutorial.